### PR TITLE
Convert class variable declarations form "var" to "public"

### PR DESCRIPTION
**Pull request by samhotchkiss**
This is a continuation of @mospaw's #2098, but moved over to be local instead of a remote PR.

Original description:
```Replacing a deprecated definition from PHP4.```

### CODE REVIEW

**Comment by samhotchkiss** [comment]
A little more background:
http://stackoverflow.com/questions/1206105/what-does-php-keyword-var-do

The `var` declaration was deprecated in PHP5, and generates an `E_STRICT` warning.  However, in 5.1.3, it was "un-deprecated" and made an acceptable synonym for `public`

I've pinged some other Automattic devs for their take on this switch.  @bryceadams let me know that WooCommerce switched to `public` a while back

**Comment by bryceadams** [comment]
For reference https://github.com/woothemes/woocommerce/pull/6963 cc @samhotchkiss

**Comment by samhotchkiss** [comment]
Got a couple more votes for `public`, so let's go with it!

File: modules/sharedaddy/sharing-sources.php [hunk 1]
```diff
@@ -1227,7 +1228,7 @@ public function display_footer() {
 		<?php if ( $this->smart ) : ?>
 			<script type="text/javascript">
 				// Pinterest shared resources
-				var s = document.createElement("script");
+				public s = document.createElement("script");
```
Comment (dereksmart) [inline modules/sharedaddy/sharing-sources.php]: looks like a search-replace snafu

**Comment by dereksmart** [comment]
LGTM.  :+1: :100: :shipit: 

God speed to all the other branches that will need a rebase after this!

**Comment by kraftbj** [comment]
Is there any harm in waiting to merge until after all of the 3.7 branches are merged in to not slow down progress on that release on rebasing and dealing with all of that?

**Comment by dereksmart** [comment]
@kraftbj I think that's a solid plan.  It's either rebase a bunch of other 3.7 branches, or rebase this one once.  

We could go one step further and squash this branch so there is only one round of conflicts.

**Comment by samhotchkiss** [comment]
Okay, so this does need to get merged before we ship beta, but it should be one of the last things to get merged.

**Comment by mospaw** [comment]
I'm glad to see this getting merged in. Thanks for the hard work getting that done @samhotchkiss

**Pull request merged**